gofannon/open_notify_space/__init__.py

```diff
@@ -0,0 +1 @@
+from . import iss_locator
```

Comment:
Drop this line, @rawkintrevo make an issue to document why this shouldn't be here.